### PR TITLE
fix(mysql): transpile CHAR

### PR DESCRIPTION
**Pull request by barakalon**
Not sure the best way to approach this one.

Sometimes its only `CHR`: 
https://www.postgresql.org/docs/9.1/functions-string.html

Sometimes its either `CHR` or `CHAR`:
https://docs.databricks.com/en/sql/language-manual/functions/chr.html

Sometimes its only `CHAR`:
https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char

And in the case of MySQL, it supports a few extra arguments. I guess we could make this a separate expression type... but that doesn't feel quite right, since in the single argument form (the most standard), they're the same.

### CODE REVIEW

File: sqlglot/expressions.py [hunk 1]
```diff
@@ -4293,6 +4293,12 @@ class Coalesce(Func):
     _sql_names = ["COALESCE", "IFNULL", "NVL"]
 
 
+class Chr(Func):
```
Comment (tobymao) [inline sqlglot/expressions.py:4296]: default should be Char

Comment (tobymao) [reply sqlglot/expressions.py:4296]: nvm i guess if postgres and databricks like chr, i'm ok with it

**Review by tobymao — APPROVED** [review]

File: sqlglot/dialects/mysql.py [hunk 2]
```diff
@@ -717,3 +730,9 @@ def _oldstyle_limit_sql(self, expression: exp.Show) -> str:
                 limit_offset = f"{offset}, {limit}" if offset else limit
                 return f" LIMIT {limit_offset}"
             return ""
+
+        def chr_sql(self, expression: exp.Chr) -> str:
+            this = self.expressions(sqls=[expression.this] + expression.expressions)
```
Comment (GeorgeSittas) [inline sqlglot/dialects/mysql.py:735]: Isn't sqls` supposed to be a list of strings?

Comment (tobymao) [reply sqlglot/dialects/mysql.py:735]: i think the type hint is wrong in expressions, it can be exp or str

**Pull request merged**